WORKSPACE

```diff
@@ -137,10 +137,9 @@ http_file(
 
 http_file(
     name = "virtio_win_image",
-    sha256 = "d5b5739cf297f0538d263e30678d5a09bba470a7c6bcbd8dff74e44153f16549",
```

Comment:
Can you provide one?

Reply:
main doesn't have one

Reply:
think it's intended? cc @ksimon1

Reply:
its available in the url if you just want to take it from there 
`57b0f6dc8dc92dc2ae8621f8b1bfbd8a873de9bedc788c4c4b305ea28acc77cd`

Reply:
https://github.com/kubevirt/kubevirt/pull/13987#issuecomment-2666192122

Reply:
Isn't the sha256 required otherwise the mirror uploader will fail? https://github.com/kubevirt/kubevirt/commit/3e767a5b5f1d2e790514fd5c343cc7cd7a513768

Reply:
right.. I guess it got added later
anyway I think we can keep this branch as is, just correct the existing mirror which is a 404